test/constraint/spec.test.ts

```diff
@@ -532,6 +532,40 @@ describe('constraints/spec', () => {
     });
   });
 
+  describe('omitAggregatePlotWithoutDimension', () => {
+    const CONFIG_WITH_OMIT_AGGREGATE_PLOT_WITHOUT_DIMENSION = extend({}, DEFAULT_QUERY_CONFIG, {omitAggregatePlotWithoutDimension: true});
```

Comment:
Please also remove similar things from your old `omitRaw` and `omitAggregate` tests.